src/App.tsx

```diff
@@ -1,10 +1,43 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { getTodos } from './api/todos';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { ErrorMessage } from './components/ErrorMessage';
+import { ApiError } from './types/ApiError';
+import { filterTodos, FilterType } from './helpers/filterTodos';
 
-const USER_ID = 0;
+const USER_ID = 6950;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [hasError, setHasError] = useState(false);
```

Comment:
```suggestion
  const hasError = errorType !== AppiError.None
```
I think you don't need this hook because you can create variable 
and use  anywhere.